docs/rst/tutorials/core/deployment/keys/topic_keys.rst

```diff
@@ -0,0 +1,234 @@
+    wget -O src/left_sensor.cpp https://raw.githubusercontent.com/eProsima/vulcanexus/{DISTRO}/docs/resources/tutorials/core/deployment/keys/src/left_sensor.cpp
+    wget -O src/controller.cpp https://raw.githubusercontent.com/eProsima/vulcanexus/{DISTRO}/docs/resources/tutorials/core/deployment/keys/src/controller.cpp
+
+The resulting directory structure should be:
+
+.. code-block::
+
+    ~/vulcanexus_ws
+     ├──src
+        ├── demo_keys_cpp
+            ├── CMakeLists.txt
+            ├── README.md
+            ├── msg
+            │   └── SensorDataMsg.idl
+            ├── package.xml
+            └── src
+                ├── left_sensor.cpp
+                ├── right_sensor.cpp
+                └── controller.cpp
+
+Building the demo package
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the environment has been setup and the demo package sources are available, we are ready to build the workspace.
+Get into the root of the workspace and build it with the following commands:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/{DISTRO}/setup.bash
+    cd ~/vulcanexus_ws
+    colcon build
+
+
+Running the demo
+----------------
+
+Finally, we can run the demo by executing the following commands in separate terminals:
+
+.. tabs::
+
+    .. tab:: Shell 1 (Right Sensor)
+
+        .. code-block:: bash
+
+            source ~/vulcanexus_ws/install/setup.bash
+            ros2 run demo_keys_cpp right_sensor
+
+    .. tab:: Shell 2 (Left Sensor)
+
+        .. code-block:: bash
+
+            source ~/vulcanexus_ws/install/setup.bash
+            ros2 run demo_keys_cpp left_sensor
+
+    .. tab:: Shell 3 (Controller)
+
+        .. code-block:: bash
+
+            source ~/vulcanexus_ws/install/setup.bash
+            ros2 run demo_keys_cpp controller
+
+.. note::
+
+    If a docker deployment was preferred, it would be necessary to attach the other two terminals to
+    the running docker container before executing the above commands.
+    This can be done by running ``docker exec -it <container_name> /bin/bash``.
+
+The resulting output should be similar to the following:
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls muted>
+        <source src="../../../../../_static/resources/tutorials/core/deployment/keys/topic_keys_tutorial.mp4">
+        Your browser does not support the video tag.
+    </video>
+
+One of the important differences to note against having used a standard topic is that, setting a reliable
+communication and defining history QoS (Quality of Service) as ``KEEP_LAST 1``, if the controller late joins the application, it is
+by no means guaranteed to receive the last data published by both of the sensors whereas by using topic keys
+we assure that the last status of each instance (sensor) is received. This is because QoS are applied per data instance.
+This and further benefits can be explored in :ref:`benefits_of_topic_keys`.
```

Comment:
We can showcase something with the history depth by having one pub publishing way faster than the other, and maybe showing that without keys a lot of samples from the slow sub are overridden

Reply:
Will be addressed in the example refactor